.github/workflows/ci.yml

```diff
@@ -24,3 +31,16 @@ jobs:
       - run: pnpm install
       - run: pnpm install typescript@${{ matrix.ts }}
       - run: pnpm type-check
+
+  are-the-types-wrong:
+    runs-on: ubuntu-latest
+    needs: run
+    steps:
+      - uses: actions/checkout@v4
+      - run: corepack enable
+      - run: pnpm install
+      - uses: actions/download-artifact@v4
+        with:
+          name: artifact
+          path: ./
+      - run: pnpx @arethetypeswrong/cli@latest ${{ needs.run.outputs.artifact }}
```

Comment:
A thought: I wonder if attw could work using a custom registry? Arguably this fits better in the pkg.pr.new workflow. 

I know you can do `attw --from-npm expect-type` but maybe it'd work with the pkg.pr.new url too. What do you think?

Reply:
Worth a try, though `pkg.pr.new` has not been kind to me.

Reply:
Oh interesting what have you seen?

Reply:
![Screenshot (80)](https://github.com/user-attachments/assets/6dfd7586-2ecc-4603-b98a-f1d595ee5279)

Reply:
I suspect I set it up in a way that makes it not-work from forked repos. (Side-note, you should have permissions to push non-protected branches to mmkal/expect-type, but probably good that you're not doing that yet to help debugging this!)

If we can get it working, I think it might be a better option. I just tried it out with the pkg.pr.new link + curl and it works v easily:

<img width="691" alt="image" src="https://github.com/user-attachments/assets/a557d76d-523c-4684-badb-5862e4a35354">

So this whole thing could reduce down to a two-liner if it's possible to fix up that pkg-pr-new workflow.

What do you think?

Reply:
Tried it out in https://github.com/mmkal/expect-type/pull/111 - can leave as a draft for now, I don't feel super strongly about which approach is better, just felt that keep the package-level testing close to where we publish a candidate package seemed right. Less worry about running `pnpm build` too many/too few times etc.

But there's not much in it!

Reply:
I'm Not sure tbh, I've tried this approach in a bunch of other repos it hasn't given me any issues but if you think #111 is a better approach I'd say let's go for it. We can always come back to this one if that one gives us any problems.

Reply:
I flip-flopped back. Closed #111: https://github.com/mmkal/expect-type/pull/108#discussion_r1722340526